src/test/java/uk/nhs/prm/deduction/e2e/tests/RepositoryE2ETests.java

```diff
@@ -61,6 +78,9 @@
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class RepositoryE2ETests {
 
+    // TODO PRMT-3252: REMOVE ONCE DONE IMPLEMENTING PRMT-3252 - FOR DEBUG USE ONLY.
+    private static final Logger debugLogger = LogManager.getLogger(RepositoryE2ETests.class);
```

Comment:
If decided to keep - constants should be uppercase 😄 

```suggestion
    private static final Logger LOGGER = LogManager.getLogger(RepositoryE2ETests.class);
```